pkg/controller/operatingsystemconfig/actuator_test.go

```diff
@@ -143,6 +145,17 @@ systemctl enable 'some-unit' && systemctl restart --no-block 'some-unit'
 		})
 
 		Describe("#Reconcile", func() {
+			It("should enable ntpd service", func() {
+				extensionConfig := Config{
+					ExtensionConfig: &v1alpha1.ExtensionConfig{UseNTP: ptr.To(true),
+						NTPConfig: &v1alpha1.NTPConfig{NTPServers: []string{"foo.bar"}}},
```

Comment:
Add a second test with multiple servers. The rendered NTP config should be checked here and in the new test.